recipe_scrapers/veroniquecloutier.py

```diff
@@ -0,0 +1,140 @@
+from collections import defaultdict
+
+from ._abstract import AbstractScraper
+from ._exceptions import FieldNotProvidedByWebsiteException
+from ._grouping_utils import IngredientGroup
+from ._utils import normalize_string
+
+
+class VeroniqueCloutier(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "veroniquecloutier.com"
+
+    def author(self):
+        return self.soup.find("strong").get_text()
+
+    def title(self):
+        return self.soup.find("h1", {"class": "title -main -page-title"}).get_text()
+
+    def total_time(self):
+        raise FieldNotProvidedByWebsiteException(return_value=None)
+
+    def yields(self):
+        potion_line = self.soup.find(
+            string=lambda text: text
+            and ("portions" in text.lower() or "donne " in text.lower())
+        )
+
+        if not potion_line:
+            return None
+
+        parent_text = potion_line.parent.get_text() if potion_line.parent else None
+        french_numbers = {
+            "un": 1,
+            "deux": 2,
+            "trois": 3,
+            "quatre": 4,
+            "cinq": 5,
+            "six": 6,
+            "sept": 7,
+            "huit": 8,
+            "neuf": 9,
+            "dix": 10,
+            "onze": 11,
+            "douze": 12,
+            "treize": 13,
+            "quatorze": 14,
+            "quinze": 15,
+        }
+        special_cases = {"dizaine": 10, "douzaine": 12}
+
+        for word in parent_text.split():
+            word_lower = word.lower()
+            if word.isdigit():
+                return f"{word} serving" if int(word) == 1 else f"{word} servings"
+            if word_lower in french_numbers:
+                number = french_numbers[word_lower]
+                return f"{number} serving" if number == 1 else f"{number} servings"
+            if word_lower in special_cases:
+                number = special_cases[word_lower]
+                return f"{number} serving" if number == 1 else f"{number} servings"
+
+        return None
+
+    def ingredients(self):
+        start = self.soup.find(
+            string=lambda text: text and text.lower() == "ingrédients"
+        )
+        end = self.soup.find(string=lambda text: text and text.lower() == "préparation")
+
+        if not start or not end:
+            return []
```

Comment:
Looks pretty great - thank you @shinyumh (and also @jknndy for reviewing).

One question for you both: do we need these `if not  start ...:` conditions?

Explanation: if something doesn't contain either any ingredients, or doesn't contain any instructions, I'd generally be skeptical of whether it's a recipe -- and so I ideally perhaps we could remove this (but I'd like to check before suggesting that).

Reply:
That is a good point. I just reviewed ~10 recipes on the website and it seemed like all of them have "Ingrédients" and "Préparation" section headers. I am open to removing these conditions if @jknndy also agrees.

Reply:
Agreed! Thanks @jayaddison

Reply:
Fixed, thank you both!